toolkit.rb

```diff
@@ -31,6 +29,11 @@
   args[arg_name] = arg_value
 end
 
+# constant set for graphql which is used in github dependabot connector
```

Comment:
no change should be needed to this file

Reply:
@imishraSecurview

Reply:
@lindambrown here i am looking for github_access_token, that psss from command line. the graphql configuration defined with constant, so when i run toolkit command from cli. the graphql configuration execute before run(opts) method. hence i have added github_access_token constant in toolkit.rb .